.storybook/main.ts

```diff
@@ -14,5 +26,14 @@ const config: StorybookConfig = {
   docs: {
     autodocs: "tag",
   },
-};
-export default config;
+  webpackFinal: async (config) => {
```

Comment:
I have no problem with that, but the docs on the new version 7.3 specify that the webpack configuration is fail proof and allow us to backtrack and divide our storybook from our original project if that's the case.

The tsconfig change is indeed simpler but I think we could have issues on updates as this seems to be a debate inside of SB team.

I've also seen a slight delay on the rendering from the logo while building the theme with the tsconfig fix, can you try this branch locally and confirm that?

Reply:
I totally get the point on the potential future separation from our main project. I'm closing the PR since we're keeping the webpack config.

I ran some tests and noticed a similar delay – could be SVG rendering or even something related to caching. I found that the delay was longer in the first render - with both approaches (webpack and tsconfig).

Your insights have given me a new way of looking at the project. Thanks for that!